docs/data/material/components/click-away-listener/click-away-listener.md

```diff
@@ -7,63 +7,19 @@ githubLabel: 'component: ClickAwayListener'
 
 # Click-Away Listener
 
-<p class="description">The ClickAwayListener component detects when a click event happens outside of its child element.</p>
+<p class="description">The Click-Away Listener component detects when a click event happens outside of its child element.</p>
 
-- 📦 [992 B gzipped](/size-snapshot/).
-- ⚛️ Supports portals
-
-{{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
-
-## Example
-
-For instance, if you need to hide a menu dropdown when people click anywhere else on your page:
-
-{{"demo": "ClickAway.js"}}
-
-Notice that the component only accepts one child element.
-You can find a more advanced demo on the [Menu documentation section](/material-ui/react-menu/#menulist-composition).
-
-## Portal
-
-The following demo uses [`Portal`](/material-ui/react-portal/) to render the dropdown into a new "subtree" outside of current DOM hierarchy.
-
-{{"demo": "PortalClickAway.js"}}
-
-## Leading edge
-
-By default, the component responds to the trailing events (click + touch end).
-However, you can configure it to respond to the leading events (mouse down + touch start).
-
-{{"demo": "LeadingClickAway.js"}}
+## Migrate to MUI Base
 
 :::warning
-⚠️ In this mode, only interactions on the scrollbar of the document is ignored.
-:::
-
-## Accessibility
-
-By default `<ClickAwayListener />` will add an `onClick` handler to its children.
-This can result in e.g. screen readers announcing the children as clickable.
-However, the purpose of the `onClick` handler is not to make `children` interactive.
-
-In order to prevent screen readers from marking non-interactive children as "clickable" add `role="presentation"` to the immediate children:
-
-```tsx
-<ClickAwayListener>
-  <div role="presentation">
-    <h1>non-interactive heading</h1>
-  </div>
-</ClickAwayListener>
-```
+Click-Away Listener is now part of the standalone [MUI Base](/base/getting-started/overview/) component library.
+It is currently re-exported in `@mui/material` for convenience, but it may be deprecated in a future major version of Material UI.
 
-This is also required to fix a quirk in NVDA when using Firefox that prevents announcement of alert messages (see [mui/material-ui#29080](https://github.com/mui/material-ui/issues/29080)).
-
-## Unstyled
-
-- 📦 [981 B gzipped](https://bundlephobia.com/package/@mui/base@latest)
-
-As the component does not have any styles, it also comes with the Base package.
+To continue using this component and avoid future breaking changes, we strongly recommend that you [install MUI Base](/base/getting-started/installation/) and import this component from the `@mui/base` package:
```

Comment:
I wouldn't put this recommendation just yet. MUI Base is still in alpha, and many organizations are reluctant to install unstable packages (even though what we reexport from Material UI is the same thing). I'd wait until beta to officially recommend this. As for now, we can point readers to the Base docs and say they can import from either package. I suppose we can mention that we plan to move it to Base completely at some point).